helix-term/src/commands.rs

```diff
@@ -2140,73 +2143,77 @@ fn global_search(cx: &mut Context) {
-                                    Some(entry) if entry.is_file() => {}
-                                    // skip everything else
-                                    _ => return WalkState::Continue,
-                                };
-
-                                let mut stop = false;
-                                let sink = sinks::UTF8(|line_num, _| {
-                                    stop = injector
-                                        .push(FileResult::new(entry.path(), line_num as usize - 1))
-                                        .is_err();
-
-                                    Ok(!stop)
-                                });
-                                let doc = documents.iter().find(|&(doc_path, _)| {
-                                    doc_path
-                                        .as_ref()
-                                        .map_or(false, |doc_path| doc_path == entry.path())
-                                });
-
-                                let result = if let Some((_, doc)) = doc {
-                                    // there is already a buffer for this file
-                                    // search the buffer instead of the file because it's faster
-                                    // and captures new edits without requiring a save
-                                    if searcher.multi_line_with_matcher(&matcher) {
-                                        // in this case a continous buffer is required
-                                        // convert the rope to a string
-                                        let text = doc.to_string();
-                                        searcher.search_slice(&matcher, text.as_bytes(), sink)
-                                    } else {
-                                        searcher.search_reader(
-                                            &matcher,
-                                            RopeReader::new(doc.slice(..)),
-                                            sink,
-                                        )
-                                    }
-                                } else {
-                                    searcher.search_path(&matcher, entry.path(), sink)
-                                };
+                        });
 
-                                if let Err(err) = result {
-                                    log::error!(
-                                        "Global search error: {}, {}",
-                                        entry.path().display(),
-                                        err
-                                    );
-                                }
-                                if stop {
-                                    WalkState::Quit
+                    walk_builder.add_custom_ignore_filename(".helix/ignore");
+                    walk_builder
+                        .add_custom_ignore_filename(helix_loader::config_dir().join(".ignore"));
```

Comment:
```suggestion
                        .add_custom_ignore_filename(helix_loader::config_dir().join("ignore"));
```

to match the one under `.helix`

Reply:
I opted to make them both `.ignore` instead because `.ignore` seems to be more common

Reply:
I would prefer `ignore` - there's no need for the file to be hidden since it will always be under a hidden directory (`.config` or `.helix`)